1_OCaml/TP/TP1.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>L'idée était plutôt d'écrire :</p><p><code></code><code>ocaml</p><p>let tanha x =&nbsp;</p><p>let e = exp x in</p><p>(e -. 1./.e) /. (e +. 1/.e));;</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>